scripts/openwebnet-discovery.nse

```diff
@@ -0,0 +1,200 @@
+local stdnse = require "stdnse"
+local shortport = require "shortport"
+local comm = require "comm"
+local string = require "string"
+
+description = [[
+OpenWebNet is a communications protocol developed by Bticino since 2000.
+Retrieves the Gateway and device type. Retrieves the count and addresses
+of lights, multimedia and many other services running on server/servers.
+]]
+
+---
+-- @usage
+-- nmap --script openwebnet-discovery
+--
+-- @output
+--  | openwebnet-discover:
+--  |   MAC address: 0-3-80-1-211-17
+--  |   Kernel Version: 2.3.8
+--  |   Net mask: 255.255.255.0
+--  |   IP address: 192.168.200.35
+--  |   Time: 19:58:33:001
+--  |   Date: 24.06.2017
+--  |   Device Type: F453AV
+--  |   Distribution Version: 3.0.1
+--  |   Firmware version: 3.0.14
+--  |   Uptime: 5.3.28.38
```

Comment:
Uptime should be reported in the same format that [stdnse.format_difftime](https://nmap.org/nsedoc/lib/stdnse.html#format_difftime) uses. You don't have to use that function, though, since it's already reported broken into segments like this. Your example output should look like "5d3h28m38s". Note that the piMyHome page does not show seconds, so some devices may not report that.